app/models/monthly_statistics_report.rb

```diff
@@ -0,0 +1,27 @@
+class MonthlyStatisticsReport < ApplicationRecord
+  MISSING_VALUE = 'n/a'.freeze
+  # how should we handle version bumps?
+  VERSION = '1.0.0'.freeze
```

Comment:
Discussed: we don't think we need to worry about this until we have changes that could break the view

Reply:
Done